src/PrestaShopBundle/Command/ListCommandsAndQueriesCommand.php

```diff
@@ -53,17 +54,26 @@ public function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $commands = $this->getContainer()->getParameter('prestashop.commands_and_queries');
```

Comment:
Im not sure if there any other ways to access container definitions besides that

Reply:
That's the question I wonder as well, I'd say it's possible since `findTaggedServiceIds` and `getDefinition` are public class but only in `ContainerBuilder` they are not in the `ContainerInterface`

And I'm not sure the compiled container has access to all this info. I'll try and test this on another occasion